bsv/MemWriteEngine.bsv

```diff
@@ -105,8 +105,14 @@ module mkMemWriteChannel#(Integer bufferSizeBytes, Integer channelNumber,
       if (clientBursts) begin
 	 load_in_progress <= True;
 	 let cmd = clientStart;
-	 let cond0 <- clientAvail.maybeDecrement(unpack(extend(cmd.burstLen>>beat_shift)));
 	 let cond1 = cmd.len <= extend(cmd.burstLen);
+         Bool cond0 = False;
```

Comment:
oh, this is a problem. maybe. I'm worried about the critical path. But better to have a long critical path than incorrect operation.